index.js

```diff
@@ -1,7 +1,7 @@
 /** @babel */
 const lazyReq = require('lazy-req')(require);
 
-const editorconfig = lazyReq('editorconfig');
```

Comment:
It seems that lazyReq is somehow too lazy for Atom in the dev-mode. Some tests fail if lazyReq is in use, running Atom in dev-mode (`atom -d`) raises an exception (~`editorconfig.parse is no function`).